File: bruteforce/hamming_distance.py
Hamming distance is the number of characters that differ between two strings.

To illustrate:

String1: "abcbba"
String2: "abcbda"

Hamming Distance: 1 - "b" vs. "d" is the only difference.
Create a function that computes the hamming distance between two strings.

def hamming_distance(txt1, txt2):
	count = 0
	for i in range(len(txt1)):
		if txt1[i] != txt2[i]:
			count += 1
	return count

File: C++/concepts/intro/hammingDistance.cpp
int hammingDistance(std::string str1, std::string str2) {
	int distance = 0;
	for(int i = 0; i < str1.length(); i++) {
		if (str1[i] != str2[i]) {
			distance += 1;
		}
	}
	return distance;
}
